scaleapi/__init__.py

```diff
@@ -114,6 +115,22 @@ def clear_task_unique_id(self, task_id: str) -> Task:
         endpoint = f"task/{task_id}/unique_id"
         return Task(self.api.delete_request(endpoint), self)
 
+    def set_task_metadata(self, task_id: str, metadata: json) -> Task:
```

Comment:
Just thinking about whether we should expect a JSON or a Dict/object as input?

Reply:
Revised to expect Dict instead of JSON